net/tstun/ifstatus_windows.go

```diff
@@ -93,7 +93,6 @@ func waitInterfaceUp(iface tun.Device, timeout time.Duration, logf logger.Logf)
 			iw.logf("TUN interface is up after %v", time.Since(t0))
 			return nil
 		case <-ticker.C:
-			break
```

Comment:
I imagine this was intentional for style; empty blocks look like an incomplete code. But sure.